src/test/kotlin/tornadofx/testapps/GotoViewApp.kt

```diff
@@ -10,11 +10,13 @@ class ContainerView : View("ContainerView") {
 
     override val root = borderpane {
         top {
-            button("Switch view").setOnAction {
-                if (center.lookup("#view1") != null)
-                    subView1.replaceWith(subView2, ViewTransition.Slide(0.2.seconds))
-                else
-                    subView2.replaceWith(subView1, ViewTransition.Slide(0.2.seconds, ViewTransition.Direction.RIGHT))
+            button("Switch view") {
```

Comment:
Okay that makes sense to me, but then we should change the other test to be consistent, like in `EventBusTest.kt`
where we have the following code:
```kotlin
     button("Load data") {
             action {
                  fire(GiveMeData)
              }
          }
```
and change it to
```kotlin
     button("Load data").action {
                  fire(GiveMeData)
     }
```
I will go through the tests and make them consistent.